yandextank/plugins/Pandora/plugin.py

```diff
@@ -134,6 +135,16 @@ def end_test(self, retcode):
             logger.debug("Seems subprocess finished OK")
         return retcode
 
+    @staticmethod
+    def patch_config(config):
+        """
+        download remote resources, replace links with local filenames
+        :param dict config: pandora config
+        """
+        ammo_location = config['pools']['ammo']['file']
```

Comment:
- ammo location is not always present in config file: https://github.com/yandex/pandora/blob/develop/example/example.yaml
- config['pools'] is a list of pools